src/main.cpp

```diff
@@ -211,6 +211,7 @@ static void lPrintVersion() {
     printf("    [--[no-]discard-value-names]\tDo not discard/Discard value names when generating LLVM IR\n");
     printf("    [--dump-file[=<path>]]\t\tDump module IR to file(s) in "
            "current directory, or to <path> if specified\n");
+    printf("    [--gen-stdlib]\t\tEnable special compilation mode to generate LLVM IR for stdlib.ispc.");
```

Comment:
This is still relevant. It would be good to issue a warning and stop compilation when `gen-stdlib` is used with input ispc file. Now this results in assertion: `ispc --gen-stdlib test.ispc`